### PR TITLE
Docs: Document 'write.parquet.row-group-check-min-record-count' and '…

### PR DESCRIPTION
**Pull request by zhongyujiang**
…write.parquet.row-group-check-max-record-count' properties.

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -58,6 +58,8 @@ Iceberg tables support table properties to configure table behavior, like the de
 | write.parquet.compression-level    | null               | Parquet compression level                          |
 | write.parquet.bloom-filter-enabled.column.col1          | (not set) | Enables writing a bloom filter for the column: col1|
 | write.parquet.bloom-filter-max-bytes | 1048576 (1 MB)   | The maximum number of bytes for a bloom filter bitset |
+| write.parquet.row-group-check-min-record-count | 100    | The minimum count of rows of checks of the page size and row group size |
```
Comment (ajantha-bhat) [inline docs/configuration.md:61]:
```suggestion
| write.parquet.row-group-check-min-record-count | 100    | The minimum record count for page size check and row group size check|
```

File: docs/configuration.md [hunk 2]
```diff
@@ -58,6 +58,8 @@ Iceberg tables support table properties to configure table behavior, like the de
...
+| write.parquet.row-group-check-max-record-count | 10000  | The maximum count of rows of checks of the page size and row group size |
```
Comment (ajantha-bhat) [inline docs/configuration.md:62]:
```suggestion
| write.parquet.row-group-check-max-record-count | 10000  | The maximum record count for page size check and row group size check|
```

**Comment by ajantha-bhat** [comment]
also cc: @stevenzwu

File: docs/configuration.md (hunk 1 above)
Comment (zhongyujiang) [reply docs/configuration.md]: Fixed.

File: docs/configuration.md (hunk 2 above)
Comment (zhongyujiang) [reply docs/configuration.md]: Fixed.

**Review by ajantha-bhat — APPROVED** [review]

**Comment by zhongyujiang** [comment]
@stevenzwu Could you please help review this?

**Pull request closed without merging**